cockroachcloud/backups-page.md

```diff
@@ -37,14 +37,14 @@ A list of your full cluster backups displays on your cluster's **Backups** page.
 For each backup, the following details display:
 
 <div class="filter-content" markdown="1" data-scope="dedicated">
-- The date and time the backup was taken (**Data From**)
-- The **Status** of the backup
+- The date and time the backup was taken (**Data From**) 
+- The **Status** of the backup 
 - The **Type** of backup
 - The **Size** of the backup
 - The remaining number of days the backup will be retained (**Expires In**)
 - The number of [**Databases**](#databases) included in the backup
 
-    To view the databases included in the backup, click the number in the [**Databases**](#databases) column.
+    To view the databases included in the backup, click the number in the [**Databases**](#databases) column
```

Comment:
To view the databases included in the backup **and initiate a restore**, click the number in the [**Databases**](#databases) column

Reply:
Is there a way to make this page open by default to "Dedicated" tab. Currently, it opens by default on the Serverless tab for me.

Reply:
@ashleykwtam I think as part of this PR let's leave the serverless filter in the same position, as this matches the rest of the docs in other product areas. The approach to which product should lead in these docs circumstances is a wider product question, beyond this PR. 
Liv, to directly link to the dedicated filter content for a customer, append `?filters=dedicated` to the end of the link.